data/brands/shop/electronics.json

```diff
@@ -317,6 +317,17 @@
         "shop": "electronics"
       }
     },
+    {
+      "displayName": "Eldorado (Україна)",
+      "locationSet": {"include": ["ua"]},
+      "matchNames": ["ельдорадо"],
+      "tags": {
+        "brand": "Eldorado",
+        "brand:wikidata": "Q60851865",
+        "name": "Eldorado",
```

Comment:
I'm curious that is this Eldorado brand don't use Ukranian but English in Ukraine?

Reply:
Actually many brands in Ukraine do not use Cyrillic names at all: McDonald's, Domino's, Eldorado, Rozetka, Comfy . Some of them previously used cyrillic, but now they have given up. This can be easily checked on the Internet

Eldorado website: https://eldorado.ua/uk/